content/client-lib-development-guide/features.textile

```diff
@@ -1901,6 +1917,20 @@ class HttpPaginatedResponse // RSC19b
   errorCode: Int // HP6
   errorMessage: String // HP7
   headers: Dict<String, String> // HP8
+  
+class Plugin // PC2
+  // Empty class/interface. Plugins are not expected to share any common interface.
+  // Still it is better to have a common base type for plugins in statically typed languages.
+  
+  enum PluginType
+  "vcdiff"
+
+class VCDiffDecoder
+  decode([byte] delta, [byte] base) -> [byte]
+  
+class DeltaExtras
+  from: String // the id of the message the delta was generated from
+  format: String //the delta format. For 1.2  only vcdiff is supported
```

Comment:
Inconsistent spacing.  
Why mention 1.2 only?  This is the 1.2 spec.  Perhaps just "Only "vcdiff" is currently supported"